src/lib/Libnet/net_server.c

```diff
@@ -528,47 +561,64 @@ wait_request(time_t waittime)
 			return (-1);
 		}
 	} else {
+		sched_processed = 0;
+		if (scks && scks->active_socks) {
+			fd_set		fdset;
+			struct timeval  timeout;
+			timeout.tv_usec = 0;
+			timeout.tv_sec = 0;
+			FD_ZERO(&fdset);
+                        for (i = 0; i < scks->active_socks; i++) {
+                                if (scks->socket_fd[i] != 0) {
+					FD_SET(scks->socket_fd[i], &fdset);
+				}
+			}
+			if (select(FD_SETSIZE, &fdset, NULL, NULL, &timeout) != -1) {
+				for (i = 0; i < scks->active_socks; i++) {
+				 	if (FD_ISSET(scks->socket_fd[i], &fdset)) {
+						log_event(PBSEVENT_DEBUG3, PBS_EVENTCLASS_SERVER,
+							LOG_DEBUG, __func__, "processing priority sockets");
+						process_socket(scks->socket_fd[i]);
+						sched_processed = 1;
```

Comment:
Better to rename the variable sched_processed to something like prty_socks_processed for better readability. Priority sockets can contain any other sockets apart from Scheduler fds in future.

Reply:
done

Reply:
done